### PR TITLE
Medical, borg and misc tweaks

### PR DESCRIPTION
**Pull request by NanakoAC**
This PR is just a couple of minor things that i've been meaning to do for a while, and that didn't seem worth giving each their own

The chemistry gripper has been moved to surgeon instead of crisis borgs. The reasoning for this is twofold:

1. Surgeon borgs fill the role of medical staff who stay in medical. Crisis borgs are medical staff who go outside, retrieve patients, or treat them in the field. Crisis borgs get to play hero. Chemistry gives surgeon borgs something to do while waiting for bones to break.
2. Surgeon borgs have a high attrition rate. They're too overspecialised (while crisis borgs are too flexible), and this results in many people picking surgeon, realising how useless they are and going SSD in frustration.

Fire extinguisher removed from surgeon because firefighting is not in their job description. I'm against giving extinguishers to everyone. Crisis borg still has extinguisher because they have reasonable need, ie rescuing people from burning areas.

Cost of reclassification board almost-removed because it was dumb. Its largely just used to fix typos.
I would like to add an admin message/log to that to prevent possible abuse though, not sure how to do it.

The mapping changes are for RP, conversations between a CMO and chemist, or chef and botanist, often had lost messages because one party moved behind the fridge at the wrong time. Due to the fact that you only hear people talking if you can see them

changes: 
  - tweak: "Tweaked surgeon cyborg modules: Added chemistry gripper, removed fire extinguisher, and added soporific to their hypospray."
  - tweak: "Removed chemistry gripper from crisis borg."
  - maptweak: "The chemistry and botany fridges are now see-through, so you can more easily chat with people on the other side."
  - tweak: "Cost of cyborg renaming module vastly decreased"
  - tweak: "Renamed medical's Chemical Closet, to Chemistry Equipment Closet. Nobody ever stores chemicals in an unrefridgerated closet."
  - rscadd: "Added two boxes of empty spraybottles to the chemistry equipment closet."

### CODE REVIEW

**Comment by NanakoAC** [comment]
Marked as risk large because i think i created a new instance of windows with the mapping tweak, and i'm not certain if that was the best way to do it, or what implications there might be

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -546,6 +546,21 @@
 		new /obj/item/weapon/storage/pill_bottle( src )
 
 
+/obj/item/weapon/storage/box/spraybottles
+	name = "box of spray bottles"
+	desc = "It has pictures of spray bottles on its front."
+	New()
```
Comment (skull132) [inline code/game/objects/items/weapons/storage/boxes.dm]: Due to coding standards, can you redo this entire file to use the proper method of proc declaration?

**Comment by skull132** [comment]
You may end up having crisis borgs overly specialized as a result of this. As previously, they've also done chemistry as an aside, to prep chemicals for treatment as such. Hrm.

**Comment by NanakoAC** [comment]
RE Crisis borgs, paramedic duty for a cyborg is extremely flexible. They also often end up firefighting, fighting slimes, fending off spiders and hanging around high-action areas.  Plus with unlimited access, they can go around and make checkups on everyone in every department, fetch SSDs, investigate mysteries, etc.They get to play hero a lot, and generally have a lot more that they can do than an organic paramedic. I
I'm very certain that giving surgeon borgs chemistry instead balances them out more, crisis will still be quite favored
also diving into space for all sorts of reasons. crisis is one of the borg modules that nobody looks at twice when they do that, and EVa rescue is often their job

**Comment by skull132** [comment]
I'm kind of dubious about removing it from them, because I know that a lot of crisis borgs make their own mixes for saving people.

By chance, is there a suggestion/feedback thread for that specific change? The rest looks and sounds fine, but that shift in specific might have a negative effect on the players if executed without feedback first.

**Pull request merged**